.travis.yml

```diff
@@ -0,0 +1,9 @@
+language: java
```

Comment:
You can crib from [this](https://github.com/trellis-ldp/trellis-ontology/blob/master/.travis.yml) if you want to use Python -- either way you'll need Java.